lib/block_view_message.go

```diff
@@ -825,6 +849,71 @@ func (bav *UtxoView) _connectMessagingGroup(
 		}
 	}
 
+	// V3 Messages: MUTING and UNMUTING members
+	if blockHeight >= bav.Params.ForkHeights.DeSoV3MessagesBlockHeight {
```

Comment:
Oh you added a blockheight :) We would actually need to add a new future blockheight for your change to work

Reply:
not sure what that means, like is there an ever higher blockHeight requirement needed for this because this technically happened after V3 started?